### ISSUE TITLE
'Cannot set property 'parent' of null' error

### ISSUE THREAD

**Issue by initialdmg** [jsdoc3/jsdoc#1400]
got this error when i run jsdoc on this javascript file. i've used eslint-airbnb checked my code. this error even appears after seeting this file ignored...

error detail: 

`node.parent = parent || null;`

file: jsdoc\lib\jsdoc\src\walker.js:664

code
> some methods and functions had been deleted for privacy. please copythe code  into a file to try and thanks : ) ):

```js
/**
 * @module
 * @ignore
 */
import $ from 'jQuery';
import Load4html from './load4Html.js';
import Load4jsx from './load4jsx.js';
import { subscribes } from '../unit/eventBus.js';

const TYPE = {
    HTML: 'html',
    JSX: 'jsx',
};
const DEFAULT_WIDTH = 300;
const loadManagerClassname = 'loadManager';
const _tools = new WeakMap();


class LoadManager {
    _$container;

    constructor(domId) {
        this._$container = $(`#${domId}`);
        if (this.$con.length === 0) {
            throw new Error(`当前 dom 节点不存在，id 为${domId}`);
        }
        _tools.set(this, []);
        _regedit.call(this);
    }
    load(tool) {
        const $con = _createToolContainer.call(this, tool.params);
        const Manager = _getManager(tool);
        Manager.load($con[0], tool);

        tool.$el = $con;
        const tmpArr = _tools.get(this);
        tmpArr.push(tool);
        _tools.set(this, tmpArr);
    }

    clear() {
        _tools.set(this, []);
    }
}

export default LoadManager;
```

**Comment by hegemonic** [comment]
Looks like this happens whenever there's a class property with no value assigned to it. This example triggers the same bug:

```js
class Foo {
    bar;
}
```

Fix coming up shortly.

**Comment by hegemonic** [comment]
Fixed on master and releases/3.5. The fix will be included in JSDoc 3.5.3.

**Comment by initialdmg** [comment]
Assigned a null to that property and everything goes well. Thanks!
